test/adapter.spec.js

```diff
@@ -367,12 +367,17 @@ describe('jasmine adapter', function () {
     })
 
     it('should set random order', function () {
-      jasmineConfig.random = true
+      jasmineConfig.random = false
       spyOn(jasmineEnv, 'randomizeTests')
 
       createStartFn(tc, jasmineEnv)()
 
-      expect(jasmineEnv.randomizeTests).toHaveBeenCalledWith(true)
+      if (jasmineEnv.configuration) {
```

Comment:
Branching in tests is a sign of trouble. Please add a comment explaining why this is needed.

Reply:
I chose to branch in the tests to work against the live versions of Jasmine. I do see that this kind of testing is more integration than unit testing.

If you would prefer I can mock out the two versions of Jasmine for testing purposes. This would lose out on potential advance warnings of future braking changes from Jasmine.